low-level/framework/sspl_ll_d

```diff
@@ -740,6 +738,26 @@ def refresh_iem_log_file(signal_number, frame):
     else:
         logger.info(f"IEMSensor is not in running, ignoring signal to refresh file")
 
+def generate_cov_report(signal_number, frame):
+    # co object will be available on execution of
+    # /opt/seagate/cortx/sspl/sspl_test/start_sspl_coverge.sh
+    logger.info("Ending Coverage Scope")
+    co.stop()
```

Comment:
Is it a global variable that coverage.py sets?

Reply:
yes, it sets a global variable.

Reply:
Okay